### ISSUE TITLE
enum_defs.json is missing values for enums with explicit values

### ISSUE THREAD

**Issue by autarch** [pganalyze/libpg_query#71]
The example I saw is for the `FunctionParameterMode` enum, which is defined in `src/postgres/include/nodes/parsenodes.h` this way:

```c
typedef enum FunctionParameterMode
{
	/* the assigned enum values appear in pg_proc, don't change 'em! */
	FUNC_PARAM_IN = 'i',		/* input only */
	FUNC_PARAM_OUT = 'o',		/* output only */
	FUNC_PARAM_INOUT = 'b',		/* both */
	FUNC_PARAM_VARIADIC = 'v',	/* variadic (always input) */
	FUNC_PARAM_TABLE = 't'		/* table function output column */
} FunctionParameterMode;
```

The JSON output for parsing will be something like `"mode": 105` where `105` is the ASCII value for `'i'`. Anything that's attempting to turn this JSON back into a language's representation (Rust in my case) will blow up here, because it's expecting these to be 0-4.

**Comment by lfittl** [comment]
Good catch - this has been resolved in the v13 branch.

**Comment by lfittl** [comment]
Closing this since the v13 branch has been released (as pg_query 2.0)
